index.html

```diff
@@ -4,6 +4,13 @@
     <title>MetaMask &lt; = &gt; Ledger Bridge</title>
   </head>
   <body>
-    <script src="./bundle.js" type="text/javascript"></script>
+    <script type="text/javascript">
+      (() => {
+        const script = document.createElement('script');
```

Comment:
This was just for debugging purposes I assume?

Reply:
Yep!